### ISSUE TITLE
Old error msg "No such package/stack" when roscd failed to find the given pkg

### ISSUE THREAD

**Issue by 130s** [ros/ros_comm#38]
In /opt/ros/groovy/share/rosbash/rosbash, following line should be changed:

```
     echo "roscd: No such package/stack '$1'"
```

**Comment by dirk-thomas** [comment]
To what should that be changed? This seems to be still a valid error message when used together with dry packages.

**Comment by 130s** [comment]
I see. I close this then.
I thought we needed to remove `stack` expression if all `stack`s would be removed right away, but now I understand `stack` still remains in `Groovy`.

p.s. Reason why I got confused in this way is the tutorial[1] says:

```
Note: rosbuild users might be wondering where stacks went, the concept of stacks was removed with catkin to simplify the growing code base and to support better distribution of packages.
```

[1] http://mirror.umd.edu/roswiki/ROS%282f%29Tutorials%282f%29NavigatingTheFilesystem.html (mirror b/c seems ros.org is down for me)
